### ISSUE TITLE
One account can't find any profiles anymore, another can

### ISSUE THREAD

**Issue by stevesbrain** [instaloader/instaloader#381]
Hey all, I have two test accounts. One test account can successfully download anything - other comes up with the following (tested against the same profile), even when it's a profile I've been able to download from that account before (or, a public profile):

```
$ instaloader --login=USERNAME --sessionfile=.insta-session --fast-update PROFILE
Loaded session from .insta-session.
Logged in as USERNAME.
Trying to find profile PROFILE using its unique ID 112233445566.
Profile PROFILE has changed its name to PROFILE. #lists the same name in both fields here
Profile PROFILE does not exist.
Trying again anonymously, helps in case you are just blocked.
Warning: PROFILE will be downloaded anonymously (Profile PROFILE does not exist.).
Downloading anonymously: PROFILE
PROFILE/2019-07-07_07-23-32_UTC_profile_pic.jpg already exists
Retrieving posts from profile PROFILE.
[  1/ 18] PROFILE/2019-07-25_21-38-09_UTC_1.jpg exists PROFILE/2019-07-25_21-38-09_UTC_2.jpg exists PROFILE/2019-07-25_21-38-09_UTC_3.jpg exists PROFILE/2019-07-25_21-38-09_UTC_4.jpg exists [Caption…] unchanged json
Saved session to .insta-session.

Errors occured:
Warning: PROFILE will be downloaded anonymously (Profile PROFILE does not exist.).
```

Nothing updates successfully, even when dropping the session file off and starting from scratch. Any clues?

**Comment by neeos** [comment]
can confirm the bug. Have been like this for me since yesterday

**Comment by stevesbrain** [comment]
I'm curious what it could be - given the account in question is completely functional through web browser, and both accounts are connecting from the same IP (so it's not IP blocked).

**Comment by neeos** [comment]
still getting this error on 4.2.9

**Comment by stevesbrain** [comment]
As am I

**Comment by nickgibi** [comment]
same problem

**Comment by Showmaster3000** [comment]
Just hit the problem of the topic starter this evening. My batch download logged the following which doesn't point to the problem above
```
JSON Query to nicknameToLoad/: 429 Too Many Requests [retrying; skip with ^C]
```
Checking for the problem manually I received the output just like above.
I opened the Instagram app on my tablet with the account I'm using, but everything went fine there. No error or other message.
After that, I retried the manual download using instaloader and it worked. Last step: checking batch download - everything fine.
So for me, the problem was solved by just open the Instagram app with the "broken" account.

**Comment by coredumperror** [comment]
We're having a similar problem, though it even extends to `curl`ing Instagram API URLs from our cloud servers that are having issues with Instaloader. It seems like we got IP-banned by Instagram's API, as changing our NAT gateway IP seems to have fixed it.

Maybe IG thought we were making too many requests? I can't imagine why, though, as we make one API request every 15 mins.

**Comment by stevesbrain** [comment]
Hmm, I'm definitely not IP banned - one account works where one doesn't. But, they may have done combo IP with account ban? I'll have to find a new IP with that account and try.

**Comment by neeos** [comment]
@Showmaster3000

I tried all of this, still nothing.

**Comment by aandergr** [comment]
The log output looks for me like you've been blocked by PROFILE. It also appears to me, that Instaloader successfully downloads the given profile. I don't see any fatal error in your log output. The reason why you don't have that output with your other account might be that it just isn't blocked.

The message `Profile PROFILE has changed its name to PROFILE.` might be confusing. However, it stems from the fact that profiles that blocked you appear as if they'd not exist, so Instaloader does an profile-by-id lookup and then finds that same profile.

**Comment by neeos** [comment]
How do you mean blocked? Because I can search whatever private / public profiles I'm trying to download perfectly fine from the Instagram app

**Comment by Showmaster3000** [comment]
For me, it seems to work again since about yesterday. My account(s) was/were definitely not blocked. If I try the download later, some of the failed profiles work, others fail again, and profiles that loaded successfully before are suddenly broken.
I think it looks like a temporary problem at Instagram.
But actually I'm using different accounts for downloading, maybe they have played with the download limits.

**Comment by stevesbrain** [comment]
@aandergr Can confirm I can browse all profiles in question through the instagram app or web without issue. I can confirm even with a random/brand new instagram account that couldn't have blocked me, I still fail with the same error. I can also confirm that no profile downloads in the state.

**Comment by everdrone** [comment]
same issue here, can browse private profiles with that account, getting the same error message

**Comment by Naheel-Azawy** [comment]
Not sure if I'm having the same issue or not but I tried with two private accounts and this is what I get:

```
Loaded session from /tmp/.instaloader-ME/session-USERNAME.
Logged in as USERNAME.
Trying to find profile PROFILE using its unique ID 123456789.
Profile PROFILE has changed its name to PROFILE.
Profile PROFILE does not exist.
Trying again anonymously, helps in case you are just blocked.
Warning: PROFILE will be downloaded anonymously ("Profile PROFILE does not exist.").
Downloading anonymously: PROFILE
PROFILE/2019-09-21_17-35-25_UTC_profile_pic.jpg already exists
PROFILE: --login=USERNAME required.
Saved session to /tmp/.instaloader-ME/session-USERNAME.

Errors occured:
Warning: PROFILE will be downloaded anonymously ("Profile PROFILE does not exist.").
PROFILE: --login=USERNAME required.
```

**Comment by Naheel-Azawy** [comment]
Not sure if I'm having the same issue or not but I tried with two private accounts and this is what I get:

```
Loaded session from /tmp/.instaloader-ME/session-USERNAME.
Logged in as USERNAME.
Trying to find profile PROFILE using its unique ID 123456789.
Profile PROFILE has changed its name to PROFILE.
Profile PROFILE does not exist.
Trying again anonymously, helps in case you are just blocked.
Warning: PROFILE will be downloaded anonymously ("Profile PROFILE does not exist.").
Downloading anonymously: PROFILE
PROFILE/2019-09-09_09-09-09_UTC_profile_pic.jpg already exists
PROFILE: --login=USERNAME required.
Saved session to /tmp/.instaloader-ME/session-USERNAME.

Errors occured:
Warning: PROFILE will be downloaded anonymously ("Profile PROFILE does not exist.").
PROFILE: --login=USERNAME required.
```

**Comment by stevesbrain** [comment]
@Naheel-Azawy Yeah seems to be same issue, and unfortunately there seems to be no interest in fixing it!

**Comment by VasiliPupkin256** [comment]
instagram do have IP and account ban. I've run into this periodically, in my case the ban is lifted after a while.

**Comment by Smart123s** [comment]
I can't download neither private nor public profiles (of course it downloads public profiles anonymously).
I can view the profile of them on the same computer from the browser.

**Comment by aandergr** [comment]
Closing this in favor of #447.

**Comment by devaftab** [comment]
I am not able to download post or anything except profile pic form a PRIVATE INSTAGRAM PROFILE.
These problems occurs while doing that

$ instaloader --login=myUserName  profles_name                      
Loaded session from /home/kali/.config/instaloader/session-myUserName.
Logged in as myUserName
[1/1] Downloading profile profile_name
profile_name/2023-06-20_18-01-18_UTC_profile_pic.jpg already exists
profile_name: Private but not followed.
Saved session to /home/kali/.config/instaloader/session-myUserName.

Errors or warnings occurred:
profile_name: Private but not followed.
